src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -97,21 +108,29 @@ public void addPerson(Person p) {
     }
 
     /**
-     * Adds a person to the address book.
-     * The person must not already exist in the address book.
+     * Adds a group to the address book.
+     * The group must not already exist in the address book.
      */
     public void addGroup(Group g) {
         groups.add(g);
     }
 
+    /**
+     * Adds a task into a specified group to the address book.
+     * The group must already exist in the address book.
+     * The task must not already exist in the address book.
+     */
+    public void addTask(Task task, Group g) {
+        groups.getGroup(g).addTask(task);
```

Comment:
Method name?

Reply:
sorry what do u mean by this ah haha?

Reply:
Oh my bad forgot to delete this comment before posting. No need to change anything here 👍 . I was just confused because of the javadoc.

Reply:
okie haha :)